postinstall.pl

```diff
@@ -0,0 +1,23 @@
+use strict;
+use warnings;
+our ($module_name, %gconfig);
+
+do 'virtualmin-htpasswd-lib.pl';
+
+sub module_install
+{
+# Enable virtualmin-htpasswd module once
+if (&virtual_server::plugin_defined("virtualmin-nginx", "start_nginx")) {
```

Comment:
Is this `start_nginx` function significant?

Reply:
Significant? It's not more significant than any other function.. We just need to test if any of Nginx functions present, that's all. I'd prefer not to use any function name at all in `plugin_defined` to test if plugin is enabled.

Reply:
Oh you're just checking if the plugin is installed? Better to look at `@virtual_server::plugins`

Reply:
> Oh you're just checking if the plugin is installed? Better to look at `@virtual_server::plugins`

Yes, actually done yesterday in this https://github.com/virtualmin/virtualmin-htpasswd/pull/6/commits/16c5b0299a276abd1608126b589871dcf104cc56 patch but forgot to comment on it.